docs/cloudformation.json

```diff
@@ -298,6 +298,13 @@
                 "ec2:Describe*",
                 "elasticloadbalancing:*",
                 "ecs:*",
+                "ecr:GetAuthorizationToken",
+                "ecr:BatchCheckLayerAvailability",
```

Comment:
I believe anything that wants to pull from ECR using the Docker API needs `GetAuthorizationToken`, `GetDownloadUrlForLayer`, `BatchGetImage` and `BatchCheckLayerAvailability`. These are included in [AmazonEC2ContainerServiceforEC2Role](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html). `GetRepositoryPolicy`, `DescribeRepositories` and `ListImages` shouldn't be necessary, as these pertain to AWS SDK operations. I copied from [AmazonEC2ContainerRegistryReadOnly](http://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr_managed_policies.html#AmazonEC2ContainerRegistryReadOnly), which is why they're there.